kotlin-reference-server/src/main/kotlin/org/stellar/data/Config.kt

```diff
@@ -0,0 +1,20 @@
+package org.stellar.data
```

Comment:
package name: `org.stellar.reference.data`

Can you also take care of the package names of other classes?